From 530a14308fabd2c5e8f265ebfca83f02a4bd3775 Mon Sep 17 00:00:00 2001
From: Johnny Robeson <johnny@localmomentum.net>
Date: Wed, 28 Jan 2015 23:12:11 -0500
Subject: [PATCH] Fix display of -c option in hhvm man page

This is a fix for https://github.com/facebook/hhvm/issues/4689

Without it, the -c (config file) option does not show the correct
text to use multiple config files.
---
 hphp/doc/man/hhvm.1 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hphp/doc/man/hhvm.1 b/hphp/doc/man/hhvm.1
index c8d9ff3..5d52a4c 100644
--- a/hphp/doc/man/hhvm.1
+++ b/hphp/doc/man/hhvm.1
@@ -106,8 +106,8 @@ Unique identifier to compiled server code.
 .TP
 .B \-c ", " \-\-config " FILE"
 Load specified config (.hdf or .ini)
-.I FILE
-. Use multiple
+.IR FILE .
+Use multiple
 .B \-c
 to specify multiple config files.
 
